client/src/standalone/videos/embed.ts

```diff
@@ -17,6 +17,7 @@ import { PeerTubeEmbedApi } from './embed-api'
 import { TranslationsManager } from '../../assets/player/translations-manager'
 import videojs from 'video.js'
 import { VideoJSCaption } from '../../assets/player/peertube-videojs-typings'
+import { AuthUser } from '@app/core/auth/auth-user.model'
```

Comment:
Please don't import things from `app` directory, we want to keep them independent